src/api-docs/source/index.html.md

```diff
@@ -123,8 +136,26 @@ along with its index in that transaction.
 Address indexer: enabled by `--index-address=true`. Allows lookup of all transactions
 involving a certain address.
 
+## Wallet plugin
+
+The main purpose of hsd node is to store blockchain data and to exchange messages with other nodes, for example
+messages about new blocks.
+
+Strictly speaking, any additional functionality is not part of the node itself. So the wallet is implemented as hsd node plugin. 
+
+By default hsd runs with wallet plugin activated, but it can be disabled by providing command line flag <code>--no-wallet</code>
+<br>(<code>no-wallet: true</code> in [hsd.conf](#hsd-conf-and-hsw-conf-files)).
+
+Wallet runs its own server, which listens for requests.
+
+Wallet can also be run as a standalone application, connecting to local or external node:
+`wallet --node-host=127.0.0.1`
```

Comment:
hm... if hsd is installed globally the command `hs-wallet` will be installed on `$PATH` which actually executes `bin/hsw`. Try it out locally and verify for me. This is related to our chat about `hsd-cli rpc` vs `hsd-rpc`... maybe this intro section should also include a mention of `npm install -g` which puts all these commands on `$PATH`:

[package.json](https://github.com/handshake-org/hsd/blob/4c901f482fc21c71e1cd71d6861be8bdeceeb7af/package.json#L55-L63)

```
  "bin": {
    "hsd": "./bin/hsd",
    "hsd-node": "./bin/node",
    "hsd-spvnode": "./bin/spvnode",
    "hs-seeder": "./bin/hs-seeder",
    "hs-wallet": "./bin/hsw",
    "hsd-cli": "./bin/hsd-cli",
    "hsw-cli": "./bin/hsw-cli"
  },
```

Reply:
Yep, global installation works, reverted back. 
I'd say we better choose one location with comprehensive info, putting everything into docs site, so it will not reference github repo page readme.